create_admin_user.py

```
@@ -0,0 +1,68 @@
+    try:
+        # Get admin credentials from environment or prompt securely
+        username = os.getenv("ADMIN_USERNAME", "admin")
+        email = os.getenv("ADMIN_EMAIL", "admin@vanta.com")
+        password = os.getenv("ADMIN_PASSWORD")
+        
+        if not password:
+            # Prompt without echo; fail in non-interactive environments
+            try:
+                password = getpass.getpass("Enter admin password: ")
+            except Exception:
+                raise RuntimeError("ADMIN_PASSWORD not set and no TTY available for prompting")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)